extension/observer/endpoints.go

```diff
@@ -33,15 +33,18 @@ const (
 	PortType EndpointType = "port"
 	// PodType is a pod endpoint.
 	PodType EndpointType = "pod"
+	// K8sNodeType is a Kubernetes Node endpoint.
+	K8sNodeType EndpointType = "k8s.node"
```

Comment:
Do you think that `node` would be too ambiguous outside of k8s context? I'm just thinking that `node` may sound better given that we have `pod`.

Or we maybe `pod` should be renamed to `k8s.pod` later on?

Reply:
There's an existing type of `k8s.node` in the spec: https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/resource/semantic_conventions/k8s.md#node

I do wonder if the other endpoint types should be migrated, as well as their relative env attributes.

Reply:
The endpoints names probably don't necessary have to follow the semantic conventions for attribute names, but I agree it seems better for consistency. Do you mind filing an issue for renaming pod endpoint type?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6723

Reply:
Thank you!